hack/create-kubeconfig.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
```

Comment:
There are a lot of scripts in the folder. Can you rename the file to arktos-apiserver-partition-create-kubeconfig.sh or create a subdirectory for similar purpose?